### PR TITLE
improve gnu units output parsing

### PR DESCRIPTION
**Pull request by iyzana**
`units 500seconds time` is currently giving output like this `0;0;0;8;20` which is not very human friendly
this PR changes the output to `8 min + 20 sec`

i believe the other outputs are unchanged:
`units 5min` => `300 s`
`units 1foot cm` => `30.48`

also, giving negative inputs is now supported, as units interpreted them as flags before

### CODE REVIEW

File: units.py [hunk 1]
```diff
@@ -31,18 +31,24 @@
...
 def handleQuery(query):
 
     if query.isTriggered:
         args = query.string.split()
         item = Item(id='python.gnu_units', icon=icon, completion=query.rawString)
         if args:
             try:
-                item.text = sp.check_output(['units', '-t'] + query.string.split(), stderr=sp.STDOUT).decode().strip()
+                output = sp.check_output(['units', '--strict', '--one-line', '--quiet'] + query.string.split(), stderr=sp.STDOUT).decode()
+                item.text = stripPrefix(stripPrefix(output.strip(), "Definition: "), "* ")
```
Comment (succcubbus) [inline units.py]: I don't really know python, so I'm not sure if this is idiomatic.

**Comment by ManuelSchneid3r** [comment]
Unfortunately I let you all wait pretty long. Now I have the problem that I don't know which PR's are valid at all. Therefore I will close all PR's for tentatively. If you want me to reopen the PR let me know. Also it would be nice if you could resolve potential conflicts due to changes in the repo. Sorry for the inconveniences, but I am working on it for a week now. I'm in a hurry. Note that this is a copy paste message. 😉

**Comment by iyzana** [comment]
No problem.

> If you want me to reopen the PR let me know.
Yep, this PR is still valid.

**Comment by ManuelSchneid3r** [comment]
Wtf

**Comment by iyzana** [comment]
That's new for me as well.
Anyway, I've opened a new pull request for it: https://github.com/albertlauncher/python/pull/91

**Pull request closed without merging**